primitives/src/extrinsics/extrinsic_params.rs

```diff
@@ -230,18 +265,16 @@ where
 /// actually contains.
 // https://github.com/paritytech/substrate/blob/1612e39131e3fe57ba4c78447fb1cbf7c4f8830e/primitives/runtime/src/generic/unchecked_extrinsic.rs#L192-L197
 #[derive(Decode, Encode, Clone, Eq, PartialEq, Debug)]
-pub struct SignedPayload<Call, SignedExtra, AdditionalSigned>(
-	(Call, SignedExtra, AdditionalSigned),
-);
+pub struct SignedPayload<Call, SignedExtra, Implicit>((Call, SignedExtra, Implicit));
```

Comment:
I think it would be nice to also rename all the `SignedExtra`-Types in generics. It is also still used in some other places (author.rs, block_builder.rs, transaction_payment.rs)

Reply:
Good catch, you're absolutely right